nixos/tests/encrypted-dns-server.nix

```diff
@@ -0,0 +1,35 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+  let
+    dnscryptServicePort = 5443;
+  in
+  {
+    name = "encrypted-dns-server";
```

Comment:
The test should be also added to the package as `passthru.tests.encrypted-dns-server`